# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../mc/ext2_hotfix.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- mc-4.1.40-pre9/vfs/undelfs.c.orig	2004-08-22 15:42:30.000000000 +0200
+++ mc-4.1.40-pre9/vfs/undelfs.c	2010-12-05 11:07:35.896055777 +0100
@@ -47,6 +47,7 @@
 #include <stdlib.h>
 /* asm/types.h defines its own umode_t :-( */
 #undef umode_t
+#include <linux/fs.h>
 #include <linux/ext2_fs.h>
 #include <ext2fs/ext2fs.h>
 #include <ctype.h>
